.github/workflows/linux.yml

```diff
@@ -45,15 +46,43 @@ jobs:
           skip_when_only_listed_labels_set: 'docs'
           skip_when_only_listed_files_changed: '*.md,*.rst,*.png,*.jpg,*.svg'
 
-  Build:
+      - name: Show affected components
+        run: |
+          echo "${{ toJSON(steps.smart_ci.outputs.affected_components) }}"
+        shell: bash
+
+  Docker:
     needs: Smart_CI
+    runs-on: aks-linux-4-cores-16gb-docker-build
+    container:
+      image: openvinogithubactions.azurecr.io/docker_build:0.2
+      volumes:
+        - /mount:/mount
+    outputs:
+      images: "${{ steps.handle_docker.outputs.images }}"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - uses: ./.github/actions/handle_docker
+        id: handle_docker
+        with:
+          images: |
+            ov_build/ubuntu_20_04_x64
+            ov_build/ubuntu_22_04_x64
```

Comment:
should we use the same OS for the whole workflow / pipeline?

Reply:
It worked: https://github.com/openvinotoolkit/openvino/actions/runs/9159853184/job/25182311114. 

@ilya-lavrenov, do you know any reason why the `ONNX_Runtime` job is using Ubuntu 22 currently?

Reply:
It was done here https://github.com/openvinotoolkit/openvino/pull/22203, but I don't remember why.
If it works on Ubuntu 20.04, then I'm OK to switch it to 20.04

But actually, we have a ticket on Jenkins CI to switch main Linux job to Ubuntu 22.04. Maybe we can do the same for GHA?

Reply:
I've created a ticket for this (141996), I guess it falls outside this PR. 

I will continue with U20 in this PR across all jobs as it works.